### PR TITLE
add: new route => "om-oss"

### PR DESCRIPTION
**Pull request by Nahom101-1**
Laget ny route for Om-oss side.

### CODE REVIEW

File: app/(site)/Om-oss/page.tsx [hunk 1]
```diff
@@ -0,0 +1,21 @@
+import BackButton from '@/components/UI/backbutton';
+import Footer from '@/components/footer/footer';
+
+export default function LesmerPage() {
+    return (
```
Comment (KjetilIN) [inline app/(site)/Om-oss/page.tsx:5]: Should be an english page name => AboutUsPage

File: package.json [hunk 2]
```diff
@@ -11,8 +11,6 @@
   "dependencies": {
     "@portabletext/react": "^3.0.4",
     "@sanity/client": "^6.6.0",
-    "@types/node": "20.8.2",
-    "@types/react": "18.2.14",
     "@types/react-dom": "18.2.6",
```
Comment (KjetilIN) [inline package.json:14]: Why this change?

File: startgjovik_website
Comment (KjetilIN) [inline startgjovik_website:1]: What is this?

**Review by KjetilIN — COMMENTED** [review]
@Nahom101-1 Added some comments.

Also you can change the button on the main page from "Les meir" to "Om oss" and make the button link to your page. 
image

**Comment by KjetilIN** [comment]
@Nahom101-1 I also see that the route name must not contain capital letters => /om-oss instead of /Om-oss

File: app/(site)/Om-oss/page.tsx (hunk 1 above)
Comment (Nahom101-1) [reply app/(site)/Om-oss/page.tsx:5]: Noted 📝

File: package.json (hunk 2 above)
Comment (Nahom101-1) [reply package.json:14]: I am not sure about this change, I didn't  manually changed it myself.

File: components/jumbotron/jumbotron.tsx [hunk 3]
```diff
@@ -12,12 +12,12 @@ export default function HeaderJumbotron() {
                         Bli med
                         <svg aria-hidden="true" className="ml-2 -mr-1 w-5 h-5" fill="currentColor" viewBox="0 0 20 20" xmlns="http://www.w3.org/2000/svg"><path fillRule="evenodd" d="M10.293 3.293a1 1 0 011.414 0l6 6a1 1 0 010 1.414l-6 6a1 1 0 01-1.414-1.414L14.586 11H3a1 1 0 110-2h11.586l-4.293-4.293a1 1 0 010-1.414z" clipRule="evenodd"></path></svg>
                     </Link>
-                    <a href="#" className="inline-flex justify-center items-center py-3 px-5 text-base font-medium text-center text-white rounded-lg border border-gray-700 hover:bg-gray-700 focus:ring-4 focus:ring-gray-800">
-                        Les mer
-                    </a>
+                    <Link href="om-oss" className="inline-flex justify-center items-center py-3 px-5 text-base font-medium text-center text-white rounded-lg border border-gray-700 hover:bg-gray-700 focus:ring-4 focus:ring-gray-800">
+                        om-oss
```
Comment (KjetilIN) [inline components/jumbotron/jumbotron.tsx:16]:
```suggestion
                        Les om oss
```
Lite forslag til en endring
